acmwebsite/templates/profile.xhtml

```diff
@@ -9,6 +9,34 @@
   </head>
 
   <body py:block="body" py:strip="True">
-    <h1 class="page-header">${u}</h1>
+    <py:import href="acmwebsite.templates.card_section" alias="cards"/>
+
+    <div class="row">
+      <!-- Header -->
+      <div class="col-md-3">
+        <img src="${u.profile_image_url}" alt="${u}" class="profile-image" />
+      </div>
+      <div class="col-md-9">
+        <h1 py:content="u" />
+        <span class="profile-tagline" py:content="u.tagline" />
+        <ul class="list-inline bold-list dot-separated-list">
+          <li>
+            ${h.icon('envelope')}
+            ${Markup(u.email_address_html)}
+          </li>
+          <li py:if="u.github_username">
+            ${h.fa_icon('github')}
```

Comment:
``h.icon`` has to go when we upgrade to Bootstrap 4. I'm also debating on whether to keep ``h.fa_icon``. It's more clear (and more consistent) if we just put in the XML markup anyway...